docs/usage/shoot-operations/shoot_credentials_rotation.md

```diff
@@ -19,6 +19,10 @@ The following steps are required to perform the rotation:
 
 1. Update the data in the `Secret` with new credentials.
 2. ⚠️ Wait until all `Shoot`s using the `Secret` are reconciled before you disable the old credentials in your cloud provider account! Otherwise, the `Shoot`s will no longer work as expected. Check out [this document](shoot_operations.md#immediate-reconciliation) to learn how to trigger a reconciliation of your `Shoot`s.
+
+    > [!TIP]
+    > If you want to verify that your new credentials are valid and usable in the Gardener environment - you can trigger [maintenance operation](shoot_operations.md#immediate-maintenance) for a shoot cluster, which will enforce infrastructure reconciliation, and wait the reconciliation to complete successfully.
```

Comment:
Not rendered as expected?

![Screenshot 2025-04-23 at 18 05 46](https://github.com/user-attachments/assets/932ca326-5d92-43b9-91ac-ce3b0f0c22d1)

Reply:
Thank you!